nyx_client/nyx_client/client.py

```diff
@@ -593,26 +606,35 @@ def create_data(
             "name": name,
             "title": title,
             "description": description,
-            "size": size,
             "genre": genre,
             "categories": categories,
             "lang": lang,
             "status": status,
             "preview": preview_base64_string,
-            "downloadURL": download_url,
             "contentType": content_type,
         }
+        if download_url:
+            data["download_url"] = download_url
+            data["size"] = (size,)
```

Comment:
Why is size a tuple?

Reply:
Ruff had done this at some stage, I've changed it back now. (Must have been doing this a funny way before)